ActivatePubKey/__tests__/handler.test.ts

```diff
@@ -344,6 +410,42 @@ describe("ActivatePubKey - Errors", () => {
     expect(popDocumentWriterMock).not.toHaveBeenCalled();
   });
 
+  it("should return 500 Internal Error when AssertionFileName fail decoding", async () => {
+    const handler = ActivatePubKeyHandler(
+      publicKeyDocumentReaderMock,
+      popDocumentWriterMock,
+      assertionWriterMock
+    );
+
+    const res = await handler(contextMock, aValidSha256AssertionRef, {
+      ...aValidPayload,
+      fiscal_code: "invalid_fiscal_code" as FiscalCode // this will make AssertionFileName decoder fail
+    });
+
+    expect(loggerMock.trackEvent).toHaveBeenCalledWith({
+      name: "lollipop.error.activate-pubkey",
+      properties: {
+        assertion_ref: aValidSha256AssertionRef,
+        message: `${FN_LOG_NAME} | Could not decode assertionFileName | value \"invalid_fiscal_code-sha256-a7qE0Y0DyqeOFFREIQSLKfu5WlbckdxVXKFasfcI-Dg\" at root is not a valid [string that matches the pattern \"^[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]-(sha256-[A-Za-z0-9-_=]{1,44}|sha384-[A-Za-z0-9-_=]{1,66}|sha512-[A-Za-z0-9-_=]{1,88})$\"]`
```

Comment:
It seems the CF can be written to logs. It's better to clean it up by removing `readableReportSimplified` details.